bigbluebutton-html5/imports/ui/services/audio-manager/index.js

```diff
@@ -283,7 +285,11 @@ class AudioManager {
   onAudioJoin() {
     this.isConnecting = false;
     this.isConnected = true;
-
+    
+    if (this.wasMuted) {
+      makeCall('toggleVoice');
```

Comment:
Is it safe to just toggle? Should we pass a desired value rather than just toggle?